src/Fhir.Anonymizer.Shared.Core/Utility/SubstituteUtility.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Fhir.Anonymizer.Core.Models;
+using Fhir.Anonymizer.Core.Processors;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Specification;
+
+namespace Fhir.Anonymizer.Core.Utility
+{
+    public class SubstituteUtility
```

Comment:
Looks like we could merge this into SubstituteProcessor? We only expect SubstituteProcessor use this, right?

Reply:
Thanks, merged this part to processor as private functions